src/backend_output_responder.cc

```diff
@@ -240,6 +346,21 @@ BackendOutputResponder::SetFixedSizeOutputBuffer(
   return cuda_copy;
 }
 
+bool
+BackendOutputResponder::SetFixedSizeStateBuffer(
```

Comment:
Do we need this function? Why not use SetFixedSizeBuffer with state flag [here](https://github.com/triton-inference-server/backend/pull/43/files#diff-84386803d308ea0441bab87c9185ef6dcff6504b1a04bb031d3102398db1c3c3R188). Instead of introducing a new function?

Reply:
I am a bit confused, why don't you call `SetFixedSizeBuffer(..., state);` directly? i.e. replacing the use of `SetFixedSizeOutputBuffer()` and `SetFixedSizeStateBuffer()` with this function